### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by jleach**
# Summary of Changes

Update many outdated packages that did not have dependency conflicts. Includes AFJ updates to latest published package.

# Related Issues

n/a

# Pull Request Checklist

This is just a reminder about the most common mistakes. Please make sure that you tick all _appropriate_ boxes. But please read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!

If an item doesn't apply to your pull request, **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this).
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components.
- [x] Run prettier: `npm run style-format`
- [x] Updated **documentation** for changed code and new or modified features.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,9 +17,9 @@
     "prepare": "husky install"
   },
   "dependencies": {
-    "@aries-framework/core": "^0.2.0-alpha.20",
-    "@aries-framework/react-hooks": "^0.1.2",
-    "@aries-framework/react-native": "^0.2.0-alpha.20",
+    "@aries-framework/core": "^0.1.0",
```
Comment (amanji) [inline package.json]: Will this not remove revocation support?

Comment (amanji) [reply package.json]: For now we may have to tolerate the alpha releases until 0.2.0

Comment (jleach) [reply package.json]: Good catch. That's a mistake. I didn't realize I went the wrong way. Will fix.

**Review by amanji — APPROVED** [review]

**Comment by jleach** [comment]
@amanji I think I'm going to have to close this PR without merging. I'm getting an issue `Property 'TextDecoder' does not exists` now - which I didn't get yesterday. I'm not sure what I've messed up. It looks like someone created this npm package `react-native-polyfill-globals` with TextDecoder in it but that just gets me onto another error `Invalid responseType: blob`.

**Pull request closed without merging**